suite/tests/linux/syscall_pwait.c

```diff
@@ -49,40 +49,73 @@
 #include <unistd.h>
 #include <time.h>
 #include <poll.h>
+#include <sched.h>
+#include "thread.h"
+#include "condvar.h"
 
 typedef struct {
     sigset_t *sigmask;
     size_t sizemask;
 } data_t;
 
+typedef struct {
+    pthread_t main_thread;
+    bool delay;
+} args_t;
+
 static struct timespec sleeptime;
```

Comment:
Optional suggestion: Making a `sleep_briefly()` or sthg function would mean this could be a local var instead of global.

Reply:
Ack